### PR TITLE
Fixes and Furry1 Beta compatch

### PR DESCRIPTION
**Pull request by TheDovahkiin97**
Do NOT merge this until I give it a go. I will check this again on HIP release.

### CODE REVIEW

File: hip-more-cultural-names/common/landed_titles/zzz_HIP_MoreCulturalNames.txt [hunk 1]
```diff
@@ -2140,12 +2140,6 @@ e_hre = {
                 }
             }
         }
-        d_osnabruck = {
-            roman = "Asenbruggensis"
```
Comment (hmlendea) [inline hip-more-cultural-names/common/landed_titles/zzz_HIP_MoreCulturalNames.txt]: Just wanted to make sure: Are these removals due to the new HIP / SWMH version introducing their own / same variants for these names?

Comment (TheDovahkiin97) [reply hip-more-cultural-names/common/landed_titles/zzz_HIP_MoreCulturalNames.txt]: they got removed

Comment (TheDovahkiin97) [reply hip-more-cultural-names/common/landed_titles/zzz_HIP_MoreCulturalNames.txt]: HIP will maybe get released tomorrow

**Comment by TheDovahkiin97** [comment]
omg you have to rename zzz_HIP_MoreCulturalNames.txt into 0_HIP_MoreCulturalNames.txt. This solves all localisation issues!

**Comment by hmlendea** [comment]
LOL

I lived with the impression that the files that are loaded last take priority. That's  why I named them zzz instad of 0.
Nice!

Also, could you please take a look at the Travis CI results? It looks like the files don't pass some of the checks.

**Comment by TheDovahkiin97** [comment]
Yeah I believe thats because of the {} blocks I deleted for the beta. my fork works perfectly with the beta unlike your version which caused mixed flags and similar. And HIP is going to be released within a few days, thats all I know.

**Comment by TheDovahkiin97** [comment]
HIP is released!
I will give this pull request a go after i checked if the release changed something in the swmh_landed_titles.txt when I get home in ~7,5 hours.
After you can safely merge and release it on steam too because the current version is not compatible with the new HIP

**Comment by TheDovahkiin97** [comment]
the pull request is safe to merge!
dont forget steamand have fun with holy fury!

**Comment by TheDovahkiin97** [comment]
ok its not bug free. My compatch causes mixed flags in italy...
ill try to find the bug

**Comment by TheDovahkiin97** [comment]
i found this little annoying bug. safe to merge now.

**Comment by hmlendea** [comment]
Ok, I will take a closer look later today.
I think I'm even gonna start a new CK2 playthrough with this opportunity :D

**Comment by TheDovahkiin97** [comment]
but dont forget to install Better Balkans & Carpathians :P

**Comment by TheDovahkiin97** [comment]
this should be all.

**Comment by hmlendea** [comment]
Do you plan any more commits or can I merge?

**Pull request merged**